packages/firebase_messaging/firebase_messaging/android/src/main/java/io/flutter/plugins/firebase/messaging/FlutterFirebaseMessagingReceiver.java

```diff
@@ -23,6 +23,9 @@ public void onReceive(Context context, Intent intent) {
       ContextHolder.setApplicationContext(context.getApplicationContext());
     }
 
+    // Pass null value to RemoteMessage constructor will lead to crash.
```

Comment:
This is fine by me as `RemoteMessage` shouldn't receive a `null` value. Could we put another debug log here to indicate no further processing? 

Something like "broadcast received but intent contained no extras to process RemoteMessage. Operation cancelled."

Reply:
Put the debug log within the `if` statement.

Reply:
Thanks for your advice, modified.